plugins/inputs/opcua_listener/README.md

```diff
@@ -48,6 +48,12 @@ to use them.
   ## Maximum time allowed to establish a connect to the endpoint.
   # connect_timeout = "10s"
   #
+  ## Behavior when we fail to connect to the endpoint on initialization. Valid options are:
+  ##     "Error" -- throws an error and exits Telegraf
+  ##     "Ignore" -- ignores this plugin, letting other plugins run
+  ##     "Retry" -- lets other plugins run, and retries connecting on every `interval`
```

Comment:
Please use lower case letters for the config options, I've also updated the descriptions:

```suggestion
  ##     "error":  throw an error and exit Telegraf
  ##     "ignore": ignore this plugin if errors are encountered
  ##     "retry":  retry connecting at each interval
```

Reply:
Done! Wasn't sure if I should try to match the other options in this plugin instead of typical TOML - this looks much better!